spmd/compiler/distribute.py

```diff
@@ -210,6 +211,7 @@ def _convert_output(
     gm: fx.GraphModule,
     node: fx.Node,
     node_to_obj: Dict[fx.Node, object],
+    logger: Optional[logging.Logger] = None,
```

Comment:
why we need to pass the logger into the function? it's already a global variable